utils/pstor/src/etcd.rs

```diff
@@ -221,23 +221,28 @@ impl StoreKv for Etcd {
         &mut self,
         key_prefix: &str,
         limit: i64,
+        range_end: &str,
     ) -> Result<Vec<(String, Value)>, Error> {
         if limit <= 2 {
             return Err(Error::PagedMinimum);
         }
 
+        let get_options = if range_end.is_empty() {
+            println!("range_end is empty");
+            GetOptions::new()
+                .with_from_key()
```

Comment:
what does this do in this case?

Reply:
If the range end is empty then it `Gets all keys >= key.`

Reply:
So for example if its `get --prefix ""` it will get us everything paginated